CONTRIBUTING.md

```diff
@@ -191,72 +193,75 @@ priority/p3 - Bugs that SHOULD be fixed by a given release, time permitting.
 
 priority/p4 - Bugs that SHOULD be fixed in a future (to be determined) release.
 
-### Backlog
+### Not Ready
```

Comment:
Ping @mhagen-vmware for reviewing these sections - these changes add details and guidelines about our pipeline lanes. The recent priority changes are unaffected.